doc/website/blog/2022-09-06-ci-for-ent.mdx

```diff
@@ -0,0 +1,307 @@
+  push:
+    branches:
+      - master
+  # Run on PRs where something changed under the `ent/migrate/migrations/` directory.
+  pull_request:
+    paths:
+      - 'ent/migrate/migrations/*'
+jobs:
+  lint:
+    services:
+      # Spin up a postgres:10 container to be used as the dev-database for analysis.
+      postgres:
+        image: postgres:10
+        env:
+          POSTGRES_DB: test
+          POSTGRES_PASSWORD: pass
+        ports:
+          - 5432:5432
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3.0.1
+        with:
+          fetch-depth: 0 # Mandatory unless "latest" is set below.
+      - uses: ariga/atlas-action@v0
+        with:
+          dir: ent/migrate/migrations
+          dir-format: golang-migrate # Or: atlas, goose, dbmate
+          dev-url: postgres://postgres:pass@localhost:5432/test?sslmode=disable
+```
+
+</TabItem>
+</Tabs>
+
+Notice that running `atlas migrate lint` requires a clean [dev-database](https://atlasgo.io/concepts/dev-database)
+which is provided by the `services` block in the example code above.
+
+## What's next for Ent CI
+
+To add to this modest beginning, I want to share some features that we are experimenting
+with at [Ariga](https://ariga.io) with hope to get the community's feedback on them.
+
+* *Linting for Online Migrations* - many Ent projects use the  automatic schema migration
+  mechanism that is available in Ent (using `ent.Schema.Create` when applications start).
+  Assuming a project's source code is managed in a version control system (such as Git),
+  we compare the schema in the mainline branch (`master`/`main`/etc.) with the one in the
+  current feature branch and use [Atlas's schema diff capability](https://atlasgo.io/declarative/diff)
+  to calculate the SQL statements that are going to be run against the database. We can then
+  use [Atlas's linting capability](https://atlasgo.io/versioned/lint) to provide insights
+  about possible dangers the arise from the proposed change.
+* *Change visualization* - to assist reviewers in understanding the impact of changes
+  proposed in a specific pull request we generate a visual diff
+  (using an ERD similar to [entviz](/blog/2021/08/26/visualizing-your-data-graph-using-entviz/)) reflect
+  the changes to a project's schema.
+* *Schema Linting* - using the official [go/analysis](https://pkg.go.dev/golang.org/x/tools/go/analysis)
+  package to create linters that analyze an Ent schema's Go code and enforce policies (such as naming
+  or indexing conventions) on the schema definition level.
+
+### Wrapping up
+
+In this post, we presented the concept of CI and discussed ways in which it
+can be practiced for Ent projects. Next, we presented CI checks we are experimenting
+with internally. If you would like to see these part of Ent or have other ideas
```

Comment:
```suggestion
with internally. If you would like to see these part of Ent or have other ideas
```
'these part of Ent'? Did you mean 'this part of Ent' or 'these parts of Ent' or 'these become a part of Ent'? :)